### ISSUE TITLE
cglobal(::Ptr)

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#10142]
the code for creating a `cglobal` from a pointer should be deleted. that operation is meaningless.

**Comment by JeffBezanson** [comment]
I agree this shouldn't exist, but I think all the code for it is shared with ccall. So fixing this might require adding code.

**Comment by vtjnash** [comment]
There's a couple cglobal-specific lines to replace with a emit_error. I just opened this as a placeholder to deal with as a cleanup later, after merging the ccall3 branch
